doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -23,7 +28,120 @@
+
+
+<a name="2.2.2"></a>
+### 2.2.2 Network Interface Options
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| n1, n2, n3, n4, n5, n6 | | Y |
+| n10, n20, n30, n40, n50, n60 | | Y |
+| n25, n50, n75, n100, n125, n150 | | Y |
+| n50, n100, n150, n200, n250, n300 | | Y |
+| n100, n200, n300, n400, n500, n600 | | Y |
+
+<a name="2.2.3"></a>
+### 2.2.3 VIM Capabilities
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.vim.cap.001 | Virtual Compute allocation | | Y |
+| e.vim.cap.002 | Virtual Storage allocation | | Y |
+| e.vim.cap.003 | Virtual Networking resources allocation | | Y |
+| e.vim.cap.004 | Multi-tenant isolation | | N <sup>1)</sup> |
+| e.vim.cap.005 | Images management | | Y |
+| e.vim.cap.006 | Virtual resources inventory per tenant | | Y |
+| e.vim.cap.007 | Resources Monitoring | | Y |
+| e.vim.cap.008 | Virtual resources Performance  |  | Y |
+| e.vim.cap.009 | Virtual resources Fault information | | Y |
+
+**1)** In a Kubernetes based infrastructure within one Kubernetes cluster multitenacy is provided only in resource management level. Isolation of execution environments requires separate Kubernetes clusters or worker nodes.
+
+<a name="2.2.4"></a>
+### 2.2.4 Instance Capabilities Mapping
+
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.nfvi.res.cap.001 | Max number of vCPU that can be assigned to a single VNFC | 16 | Y |
+| e.nfvi.res.cap.002 | Max memory in MB that can be assigned to a single VNFC by NFVI | 32 GB | Y |
+| e.nfvi.res.cap.003 | Max storage in GB that can be assigned to a single VNFC by NFVI | 320 GB | Y |
+| e.nfvi.res.cap.004 | # Connection Points | 6 | Y |
+| e.nfvi.res.cap.005 | Total instance (persistent) storage (GB | 300 GB | Y |
+| e.nfvi.per.cap.001 | CPU pinning support | | Y |
+| e.nfvi.per.cap.002 | NUMA support | | Y |
+| e.nfvi.per.cap.003 | IPSec Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.004 | Crypto Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.005 | Transcoding Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.006 | Programmable Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.007 | Enhanced Cache Management | | X (if offered |
+| e.nfvi.mon.cap.001 | Monitoring of L2-7 data | | N<sup>1)</sup> |
+| i.nfvi.sla.cap.001 | CPU overbooking | 1:1, 1:4 | Y |
+| i.nfvi.sla.cap.002 | vNIC QoS | | Y |
+| i.nfvi.per.cap.001 | Huge page support |  | Y |
+| i.nfvi.mon.cap.001 | Monitor host CPU usage |  | Y |
+| i.nfvi.mon.cap.002 | Monitor virtual compute CPU usage | | Y |
+| i.nfvi.mon.cap.003 | Monitor host CPU utilization | | Y |
+| i.nfvi.mon.cap.004 | Monitor virtual compute CPU utilization | | Y |
+| i.nfvi.mon.cap.007 | Monitor external storage capacity | | Y |
+| nfvi.com.cfg.001 | CPU allocation ratio  | 4:1, 1:1 | Y |
```

Comment:
What is the difference between i.nfvi.sla.cap.001 and nfvi.com.cfg.001?

Reply:
I see no difference, it looks like this is defined two times in the RM.

Reply:
@CsatariGergely It is defined once in the infrastructure level and another time at the virtual compute (not sure why and the ratios are inverse of each other). I think you can ignore the virual compute "nfvi.com.cfg.001." Thanks

Reply:
Removed.